src/node_crypto.cc

```diff
@@ -2911,7 +2930,8 @@ void CipherBase::SetAuthTag(const FunctionCallbackInfo<Value>& args) {
   unsigned int tag_len = Buffer::Length(args[0]);
   const int mode = EVP_CIPHER_CTX_mode(cipher->ctx_);
   if (mode == EVP_CIPH_GCM_MODE) {
```

Comment:
@tniessen I know your change is for `GCM`. I just think what's the minimal check we should do for `CCM`? Something like:
```
(cipher->auth_tag_len_ >= 0 && cipher->auth_tag_len_ != tag_len) || tag_len > EVP_GCM_TLS_TAG_LEN
```

Reply:
@yhwang This is basically the same problem as we discussed in #20040. I will fix it in that PR so this can remain semver-minor.

Reply:
yes!